src/frontends/onnx/frontend/src/frontend.cpp

```diff
@@ -135,6 +153,32 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
 void FrontEnd::add_extension(const std::shared_ptr<ov::Extension>& extension) {
     if (auto telemetry = std::dynamic_pointer_cast<TelemetryExtension>(extension)) {
         m_extensions.telemetry = telemetry;
+    } else if (auto transformation = std::dynamic_pointer_cast<DecoderTransformationExtension>(extension)) {
+        m_transformation_extensions.push_back(transformation);
+    } else if (const auto& so_ext = std::dynamic_pointer_cast<ov::detail::SOExtension>(extension)) {
+        add_extension(so_ext->extension());
+        m_other_extensions.push_back(so_ext);
+    } else if (auto common_conv_ext = std::dynamic_pointer_cast<ov::frontend::ConversionExtension>(extension)) {
+        m_conversion_extensions.push_back(common_conv_ext);
+
+        for (int i = 1; i < 13; ++i)
```

Comment:
BTW, we have a constant with a value of the highest ONNX opset that we support. You can use this constant in those for loops https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/CMakeLists.txt#L13 and it will be automatically updated in the future along with switches to a newer ONNX version.

Reply:
done